react.js

```diff
@@ -97,6 +97,7 @@ module.exports = {
     'react/no-unescaped-entities': 'warn',
     'react/no-unknown-property': 'error',
     'react/no-unsafe': 'warn', // if you need it there should be a comment explaining why
+    'react/no-unstable-nested-components': 'error',
```

Comment:
I think there are some libs I've used that don't prefix with `render`, so let's allow as props here.

```suggestion
    'react/no-unstable-nested-components': ['error', {allowAsProps: true}],
```